<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orgaSelectWidget</class>
 <widget class="QWidget" name="orgaSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitterOrga">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLineEdit" name="champRechercheOrga">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border: 2px solid gray;
padding: 0 8px;
selection-background-color: darkgray;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bouttonRechercheOrga">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Recherche</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetOrga"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>orgaSelected(orga_ptr)</signal>
 </slots>
</ui>
